docker/images/fuseki/.dockerignore

```diff
@@ -0,0 +1 @@
+t11
```

Comment:
Yeah - in the commit message "for future use" but - I can back out that commit. I added it for parallelism among the different images but that's unnecessary.